index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="styles/style.css" rel="stylesheet" />
+    <title>Saria Pastor ° A Portfolio</title>
+</head>
+<body>
+    <header>
+        <div>
+            Saria Pastor
+        </div>
```

Comment:
Could this be a h1 tag instead of a div?

Maybe your comment "the validator suggested making divs because the content was not identified by headings" was referring to this.